gef-ui/server/utils/display-items.js

```diff
@@ -109,6 +109,35 @@ const facilityItems = (facilityUrl, { type, hasBeenIssued, shouldCoverStartOnSub
       },
       isHidden: !hasBeenIssued,
     },
+    {
+      label: 'Has a facility end date',
+      id: 'facilityEndDateExists',
+      href: `${facilityUrl}/about-facility?status=change`,
+      method: (value) => (isTrueSet(value) ? BOOLEAN.YES : BOOLEAN.NO),
+      // TODO DTFS2-7171: Add logic to also hide if deal version doesn't exist or is < 1
```

Comment:
Thinking about this -- Are we going to have a minimum version number on each of these items and filter the array?

Reply:
(also, does this fall in 7171? I'd probably expect to see the logic in this ticket or have you and mark coordinated? :) )

Reply:
Now handled in this ticket!